src/theme.ts

```diff
@@ -106,4 +106,5 @@ export const theme = createTheme({
     },
     primaryShade: 5,
     primaryColor: 'purple',
+    columnWidth: '20%',
```

Comment:
what's this for 🤔 it's a global setting so just want to make sure this is very intentional and thought out

Reply:
i ended up removing this